src/components/HomeIntro/index.js

```diff
@@ -61,11 +61,7 @@ const HomeIntro = () => {
                 Learn how you can help
               </button>
             </a>
-            <div className={Styles.ctaSecondary}>
-              <a href="#" title="See how climate change happens">
-                Need convinced? Watch this
-              </a>
-            </div>
+            <ConvincedButton />
```

Comment:
I think this needs imported, which is what is causing the error at build time

Reply:
Ah, I must have accidentally removed it during a rebase. Great spot, thanks!